Makefile

```diff
@@ -1,15 +1,27 @@
-current_user   := $(shell id -u)
-current_group  := $(shell id -g)
-BUILD_DIR      := $(PWD)
-DOCKER_FLAGS   := --interactive --tty
-DOCKER_IMAGE   := registry.gitlab.com/fun-tech/fundraising-frontend-docker
-COVERAGE_FLAGS := --coverage-html coverage
+current_user    := $(shell id -u)
+current_group   := $(shell id -g)
+BUILD_DIR       := $(PWD)
+DOCKER_FLAGS    := --interactive --tty
+DOCKER_IMAGE    := registry.gitlab.com/fun-tech/fundraising-frontend-docker
+COVERAGE_FLAGS  := --coverage-html coverage
 
 install-php:
-	docker run --rm $(DOCKER_FLAGS) --volume $(BUILD_DIR):/app -w /app --volume ~/.composer:/composer --user $(current_user):$(current_group) $(DOCKER_IMAGE):composer composer install $(COMPOSER_FLAGS)
+	docker run --rm $(DOCKER_FLAGS) \
+		--volume ~/.composer:/composer \
+		--volume $(BUILD_DIR):/app \
+		-w /app \
+		--user $(current_user):$(current_group) \
+		$(DOCKER_IMAGE):latest \
+		composer install $(COMPOSER_FLAGS)
 
 update-php:
-	docker run --rm $(DOCKER_FLAGS) --volume $(BUILD_DIR):/app -w /app --volume ~/.composer:/composer --user $(current_user):$(current_group) $(DOCKER_IMAGE):composer composer update $(COMPOSER_FLAGS)
+	docker run --rm $(DOCKER_FLAGS) \
+		--volume ~/.composer:/composer \
+		--volume $(BUILD_DIR):/app \
+		-w /app \
+		--user $(current_user):$(current_group) \
+		$(DOCKER_IMAGE):latest \
```

Comment:
`$(DOCKER_IMAGE):latest` should be `$(DOCKER_IMAGE):composer` or not? (because that is how it was initially)

Reply:
The purpose of this change is to use *one* docker image for everything. "Image" in this context means a combination of repository path (`registry.gitlab.com/fun-tech/fundraising-frontend-docker`) and image tag (`:latest`). See https://medium.com/@seanoughton/docker-images-a-closer-look-ff671c303a8f

Previously, we had several docker images, `registry.gitlab.com/fun-tech/fundraising-frontend-docker:composer` for composer-related commands, `registry.gitlab.com/fun-tech/fundraising-frontend-docker:latest` for local CI tasks and `registry.gitlab.com/fun-tech/fundraising-frontend-docker:ci` for CI task in Drone CI. The new image, `registry.gitlab.com/fun-tech/fundraising-frontend-docker:latest`, now contains all the dependencies: composer, CI tools, XDebug (for gathering coverage data). I introduced the changes in https://gitlab.com/fun-tech/fundraising-frontend-docker/-/merge_requests/23 and it got merged. That means the "Use unified Docker image" PRs *should*  only use `registry.gitlab.com/fun-tech/fundraising-frontend-docker:latest`